android/beagle/src/main/java/br/com/zup/beagle/android/context/ContextDataEvaluation.kt

```diff
@@ -69,12 +67,8 @@ internal class ContextDataEvaluation(
         expression: String,
         bind: Bind.Expression<*>
     ) {
-        val value = evaluateExpression(contextData, bind.type, expression)
-        if (value != null) {
-            bind.evaluatedExpressions[expression] = value
-        } else {
-            bind.evaluatedExpressions.remove(expression)
-        }
+        val value = evaluateExpression(contextData, bind, expression)?:""
```

Comment:
A small suggestion: add spaces between operators
```suggestion
        val value = evaluateExpression(contextData, bind, expression) ?: ""
```